From 7749b67f6cff3246444051764b5bc62decfb9f58 Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Fri, 28 Aug 2015 15:01:55 +0200
Subject: [PATCH 2/3] configs: ti_armv7_common.h: Add default addresses for pxe
 and scripts

Add mandatory address variables for loading scripts and pxe configuration as
per README.distro

Signed-off-by: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
---
 include/configs/ti_armv7_common.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/ti_armv7_common.h b/include/configs/ti_armv7_common.h
index 6dc6515..586bd2b 100644
--- a/include/configs/ti_armv7_common.h
+++ b/include/configs/ti_armv7_common.h
@@ -56,6 +56,8 @@
 	"fdt_addr_r=0x88000000\0" \
 	"rdaddr=0x88080000\0" \
 	"ramdisk_addr_r=0x88080000\0" \
+	"scriptaddr=0x80000000\0" \
+	"pxefile_addr_r=0x80100000\0" \
 	"bootm_size=0x10000000\0"
 
 #define DEFAULT_MMC_TI_ARGS \
-- 
2.1.4

